SU-SWS/saa_victoria_subtheme
# Adding in Proxima Nova for Body

# READY FOR REVIEW

# Summary
- Proxima Nova is getting overridden on the Associates Designees page by system fonts.
THIS MIGHT BE THE WRONG APPROACH. If so, feel free to ignore. The goal of this PR was just to get us moving, if possible. 

# Review By (Date)
- ASAP (Tue)

# Urgency
- Relatively urgent

# Steps to Test

1. Pull down branch
2. Make sure it doesn't screw things up on normal pages
3. Push up and test on Associates page and make sure it works and that intro text is rendered in Proxima Nova


# Affected Projects or Products
- SAA Victoria Sub-Theme.

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
